### ISSUE TITLE
search for specific function in the xref window

### ISSUE THREAD

**Issue by nitrogar** [Vector35/binaryninja-api#4532]
**What is the feature you'd like to have?**
the search in the xref window only look for element's usage and not  where it is used.
for example  i want to check if a global variable is used in specific function i need to scroll the full xref windows.
but if i can for example write the function name in the search bar and filter out all other xrefs in other functions

**Is your feature request related to a problem?**
yeah, for example if i have a huge function A and i want to look for function B usage inside function A, i need to scroll down all function A. but if this feature is implemented then i can simply look for the xref for B in A

**Are any alternative solutions acceptable?**
manually scroll the xref window

**Additional Information:**
basically i want to add the callee function's name in the search list for the xref search bar. so the blue colored function name is included in the search 

![image](https://github.com/Vector35/binaryninja-api/assets/53944555/4755d71a-d60d-4a41-b0e4-fc47d7a4651f)

**Comment by nitrogar** [comment]
another picture for illustration, here i search for "sub" which removed most of the xref, even thought all function names include "sub".

![image](https://github.com/Vector35/binaryninja-api/assets/53944555/fde0efcf-e363-418f-8fc2-230acd3087d4)

**Comment by xusheng6** [comment]
How about using the full name of the function, e.g., sub_4b7607, in the filter? That should allow you to filter based on the function

**Comment by nitrogar** [comment]
it does not work :(.
![image](https://github.com/Vector35/binaryninja-api/assets/53944555/a2d63db8-c363-4b63-b757-ce516d94b414)
![image](https://github.com/Vector35/binaryninja-api/assets/53944555/1a18c42f-71a8-4ba3-80be-89250e2c4606)

**Comment by xusheng6** [comment]
> another picture for illustration, here i search for "sub" which removed most of the xref, even thought all function names include "sub".
> 
> ![image](https://user-images.githubusercontent.com/53944555/257005648-fde0efcf-e363-418f-8fc2-230acd3087d4.png)

OK, I figured out. The search in the xref window is not taking function names into consideration. It only searches the text. And in the screenshot you showed, these ones remain after the filter because they have the text `sub` in their HLIL code, not the function name

**Comment by xusheng6** [comment]
@nitrogar As a workaround, you can switch the xref widget to table mode and then search it. The search in table mode involves the function name. You can switch the xref widget to table mode by right click in the xref widget, and then select `View as table`.
